frontend/src/metabase/redux/downloads.ts

```diff
@@ -117,23 +118,33 @@ export const downloadQueryResults = createAsyncThunk(
     });
 
     if (opts.type === Urls.exportFormatPng) {
-      downloadChart(opts);
+      const isWhitelabeled = getTokenFeature(getState() as State, "whitelabel");
```

Comment:
You can import `createAsyncThunk` from `"metabase/lib/redux"` to remove the need in the type casting

Reply:
TIL
Thank you

Reply:
Will address in PRG-13